src/core/CorePool.ts

```diff
@@ -113,10 +133,22 @@ export class CorePool {
     tickUpper: number,
     amount: JSBI
   ): { amount0: JSBI; amount1: JSBI } {
-    // TODO
+    let positionStep = this.modifyPosition(owner, tickLower, tickUpper, amount);
+
+    let amount0 = JSBI.bitwiseNot(positionStep.amount0);
+    let amount1 = JSBI.bitwiseNot(positionStep.amount1);
```

Comment:
It seems `JSBI.unaryMinus` as Negation is qualified here instead of Bitwise negation.

Reply:
Thx, my misunderstand here